code/datums/supply_packs/spec_ammo.dm

```diff
@@ -184,6 +184,65 @@
 	containername = "M4RA Scout Impact Magazine Crate"
 	group = "Weapons Specialist Ammo"
 
+//SHARP
+
+/datum/supply_packs/ammo_grenader_sharp_mix
+	name = "SHARP Grenader Mixed Magazine Crate (explosive x2, flechette x2, tracking x2)"
```

Comment:
grenader is not a word, it's "grenadier", fix this throughout

Also I don't think the gun is called a grenader, should be called SHARP [numbers] rifle?

Reply:
other weapons are semi-consistent with this naming scheme (since i assume its easier for players to know their role rather than weapon names?)

Reply:
still fix grenader